trio/_core/tests/test_multierror.py

```diff
@@ -91,6 +113,52 @@ def test_MultiError():
         MultiError([KeyError(), ValueError])
 
 
+@pytest.mark.skipif(
+    sys.version_info < (3, 6, 4), reason="Unsupported in python < 3.6.4"
+)
```

Comment:
Also, I think this should actually pass on 3.5, because it's not calling the traceback logic? If so then the `skipif` should be removed.

Reply:
Actually, pytest must be calling `format_exception` implicitly or something, because even with the latest patch, this test wail on 3.5.

Now this test is actually redundant with the two others, so maybe we can just remove it entirely?

Reply:
The test actually passes on 3.5 so I removed the `skipif`